src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -220,13 +210,7 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({ author }) => {
 
   useEffect(() => {
     toggleButtonHandler();
-  }, [errorFields]);
-
-  useEffect(() => {
-    if (newAuthorValues.socialNetwork.some((network) => network != null)) {
-      setErrorFields({ ...errorFields, socialNetwoks: '' });
-    }
-  }, [newAuthorValues.socialNetwork]);
+  }, [errorMessages]);
```

Comment:
`toggleButton`(I would name it `isSaveDisabled` instead) does not need to be a state variable:

```
const isSaveDisabled = useMemo(
  () => Object.values(errorMessages).some(message => message),
  [errorMessages]
);
```